### PR TITLE
Change envconfig.Usage prefix to 'nsm'

### PR DESCRIPTION
**Pull request by arp-est**

### CODE REVIEW

**Review by denis-tingaikin — APPROVED** [review]

File: main_test.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2020-2022 Doc.ai and/or its affiliates.
+// Copyright (c) 2020-2024 Doc.ai and/or its affiliates.
 //
-// Copyright (c) 2022-2023 Cisco Systems, Inc.
+// Copyright (c) 2022-2024 Cisco Systems, Inc.
```
Comment (denis-tingaikin) [inline main_test.go:3]: These lines can be reverted.

File: pkg/imports/gen.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020-2022 Cisco and/or its affiliates.
+// Copyright (c) 2020-2024 Cisco and/or its affiliates.
```
Comment (denis-tingaikin) [inline pkg/imports/gen.go:1]: This line can be reverted.

File: main.go [hunk 3]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2020-2022 Doc.ai and/or its affiliates.
+// Copyright (c) 2020-2024 Doc.ai and/or its affiliates.
 //
-// Copyright (c) 2023 Cisco Systems, Inc.
+// Copyright (c) 2023-2024 Cisco Systems, Inc.
```
Comment (denis-tingaikin) [inline main.go:3]: The best option to edit this file is append a new copyright holder

For example,
```go
// Copyright (c) 2020-2022 Doc.ai and/or its affiliates.
//
// Copyright (c) 2023 Cisco Systems, Inc
//
// Copyright (c) 2024 ${YOUR COMPANY NAME e.g. Nordix Foundation }. 
```

**Review by denis-tingaikin — COMMENTED** [review]
Added some comments related to copyright things

File: main_test.go (hunk 1 above)
Comment (arp-est) [reply main_test.go]: Done

File: pkg/imports/gen.go (hunk 2 above)
Comment (arp-est) [reply pkg/imports/gen.go]: Done

File: main.go (hunk 3 above)
Comment (arp-est) [reply main.go]: Thanks, Done. (Apparently it changed its name from nordix to openinfra)

Comment (denis-tingaikin) [reply main.go]: Nice! Now,  license things look close to reality. Thanks!

**Pull request merged**